source/API_Reference/Web_API_v3/cancel_scheduled_send.apiblueprint

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
@john-cai I merged the two docs and put them in the v3 root for now. I'll manage moving it to the correct place when more features are available.